mobile/src/screens/chat-list/chat-list.tsx

```diff
@@ -54,7 +61,13 @@ const ChatList: React.FC = () => {
         />
         <ScrollView contentContainerStyle={styles.list}>
           {filteredChats.map((item) => (
-            <Card title={item.title} onPress={onPress} key={item.id} />
+            <Card
+              title={item.title}
+              onPress={(): void => {
+                onPress(item.title);
+              }}
```

Comment:
Avoid inline functions

Reply:
You can make currying function that returns void by calling `onPress(item.title)` and pass it there